src/network-layer/default/RelayDefaultNetworkLayer.js

```diff
@@ -163,7 +163,11 @@ function throwOnServerError(response: any): any  {
   if (response.status >= 200 && response.status < 300) {
     return response;
   } else {
-    throw response;
+    return response.text().then(text => {
```

Comment:
How would you feel about sharing the code that already handles mutation errors (see `if (payload.hasOwnProperty('errors')` of `sendMutation()`)? Perhaps we should trigger the same error handling code when `errors` is present in the payload or when the status is non-200.